app/src/main/java/com/cornellappdev/android/eatery/components/BarChartRoundedRenderer.java

```diff
@@ -0,0 +1,183 @@
+package com.cornellappdev.android.eatery.components;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+
+import com.github.mikephil.charting.animation.ChartAnimator;
```

Comment:
idk which ones you don't need, but make sure to delete any imports that you no longer need!